joda-money/src/main/java/com/fasterxml/jackson/datatype/jodamoney/AmountConverter.java

```diff
@@ -0,0 +1,13 @@
+package com.fasterxml.jackson.datatype.jodamoney;
+
+import org.joda.money.CurrencyUnit;
+import org.joda.money.Money;
+
+import java.math.BigDecimal;
+
+public interface AmountConverter<T> {
```

Comment:
Javadocs here would be nice  (although mostly if this is offered as an extension point as I suggest -- if not, just internal type)